lua/neogen/generator.lua

```diff
@@ -200,10 +247,11 @@ return setmetatable({}, {
             notify("Language " .. filetype .. " not supported.", vim.log.levels.WARN)
             return
         end
-        typ = (type(typ) ~= "string" or typ == "") and "func" or typ -- Default type
+        node_type = (type(node_type) ~= "string" or node_type == "") and "any" or node_type -- Default type
+
         local template = language.template
-        if not language.parent[typ] or not language.data[typ] or not template or not template.annotation_convention then
-            notify("Type `" .. typ .. "` not supported", vim.log.levels.WARN)
+        if not template or not template.annotation_convention then
+            notify("Type `" .. node_type .. "` not supported", vim.log.levels.WARN)
```

Comment:
The notify message here should be something like "Language does not support any annotation convention"

Reply:
I changed this to `Language " .. filetype .. " does not support any annotation convention`. Please let me know if there is a better alternative